packages/core/src/Stores/client-store.js

```diff
@@ -765,6 +766,13 @@ export default class ClientStore extends BaseStore {
         return this.account_status?.status?.some(status => status === 'trading_experience_not_complete');
     }
 
+    get is_verified() {
```

Comment:
@suisin-deriv Add the type man 👀 😬

Reply:
@farzin-deriv I will remove this code soon